WIP 3.0 Classes/Warlock.md

```diff
@@ -180,6 +188,8 @@
 
 </style>
 
+</style>
```

Comment:
Is this style tag in excess? I don't see anything corresponding that'd warrant it.

Reply:
Probably.

Reply:
I don't really mess with the style stuff, so it's most likely a remnant of a copy-paste or something.